app/views/tags/index.html.erb

```diff
@@ -0,0 +1,24 @@
+<h1>Tags List</h1>
+<table class="tags table">
+  <tbody>
+    <% @tags.each do |tag| %>
+      <tr>
+        <td>
+          <span><%= link_to tag.title, tag %></span>
```

Comment:
It looks like you might have accidentally duplicated all of these lines? They will all display the same tag. You can remove these extra lines so that it only displays the tag once.